### ISSUE TITLE
XML format: add support for multiple keys per element

### ISSUE THREAD

**Issue by kairos10** [pettarin/penelope#19]
It seems that the current parser does not support multiple keys for a single definition.

    <entry><key>form1</key><key>form2</key><definition>some definition applicable for both keys</definition></entry>

**Comment by pettarin** [comment]
Indeed. The DTD of the (Penelope-defined) XML format is https://github.com/pettarin/penelope/blob/master/penelope/res/dictionary.dtd

In view of https://github.com/pettarin/penelope#important-update I leave the issue open, but I do not plan to address it. You can workaround this issue by creating two ``<entry>`` elements with the same ``<definition>`` but different ``<key>`` values.
